src/integrationTest/integrationTestsUtilities.ts

```diff
@@ -10,11 +10,25 @@ import * as vscode from 'vscode'
 // java8.al2 image does a while to pull
 const LAMBDA_SESSION_TIMEOUT = 60000
 
+export const MAPPING_SCHEMA_TIMEOUT = 50000
+export const MAPPING_SCHEMA_INTERVAL = 2500
+
 // Retrieves CodeLenses from VS Code
 export async function getCodeLenses(uri: vscode.Uri): Promise<vscode.CodeLens[] | undefined> {
     return vscode.commands.executeCommand('vscode.executeCodeLensProvider', uri)
 }
 
+export async function getCompletionItems(
+    uri: vscode.Uri,
+    position: vscode.Position
+): Promise<vscode.CompletionList | undefined> {
+    return vscode.commands.executeCommand('vscode.executeCompletionItemProvider', uri, position)
+}
+
+export async function getHoverItems(uri: vscode.Uri, position: vscode.Position): Promise<vscode.Hover[] | undefined> {
+    return vscode.commands.executeCommand('vscode.executeHoverProvider', uri, position)
+}
```

Comment:
these and getCodeLenses() can live in `src/shared/utilities/vsCodeUtils.ts`. They are not test-specific.